src/components/DeviceListTable.tsx

```diff
@@ -47,6 +52,21 @@ const DeviceListTable: FC<Props> = ({ configBaseURL, devices }) => {
   ];
 
   const deviceRows = overviewDevices.map(([devicename, device]) => {
+    let deviceType = device.type;
+    if (isDiskDevice(device)) {
+      if (isRootDisk(device as FormDevice)) {
+        deviceType += " (root)";
+      }
+
+      if (isVolumeDevice(device)) {
+        deviceType += " (volume)";
+      }
+
+      if (isHostDiskDevice(device)) {
+        deviceType += " (host)";
```

Comment:
```suggestion
        deviceType += " (host path)";
```
Might be easier to understand. Wdyt?

Reply:
I was thinking the detail column displays a path, so seems a little redundant to have `(host path)` in the type column?

Reply:
My concern is this: Just because there is a path next to the type, it doesn't fully explain the path refers to the host. It could be understood as the mount path as well.

Reply:
Yes agreed, adding `path` :+1: